### PR TITLE
lights subsystem (v1)

### PR DESCRIPTION
**Pull request by duane701**
Please review these changes.

Please compile this branch to ensure that this code is correct. (I do not have a compiler.)

### CODE REVIEW

File: src/RobotMap.cpp [hunk 1]
```diff
@@ -75,4 +81,12 @@ void RobotMap::init() {
 
     shooterLifter.reset(new DoubleSolenoid(kLifterF, kLifterR));
     lw->AddActuator("Shooter", "Lifter", shooterLifter);
+
+    //..........lights..........
+
+    lightsFeature.reset(new Relay(0));
+    lw->AddActuator("Lights", "Feature", lightsFeature);
+
+    lightsShooter.reset(new Relay(1));
```
Comment (Diego-Andrade) [inline src/RobotMap.cpp]: Could use a constant instead of fixed number for relay ports

File: src/Subsystems/Lights.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#include "Lights.h"
+#include "../RobotMap.h"
+
+Lights::Lights()
+  : Subsystem("Lights")
+{
+  feature = RobotMap::lightsFeature;
+  shooter = RobotMap::lightsShooter;
```
Comment (Diego-Andrade) [inline src/Subsystems/Lights.cpp]: Use of member initializer list would be more consistant

File: src/Subsystems/Lights.cpp [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  SetFeatureLights(Relay::kForward);
+}
+
+void Lights::InitDefaultCommand()
+{
+  // Set the default command for a subsystem here.
+  //SetDefaultCommand(new MySpecialCommand());
```
Comment (Diego-Andrade) [inline src/Subsystems/Lights.cpp:15]: No default command?

**Review by Diego-Andrade — CHANGES_REQUESTED** [review]
Code complies as is. A few inconsistencies and missing default command

File: src/Subsystems/Lights.cpp (hunk 3 above)
Comment (duane701) [reply src/Subsystems/Lights.cpp:15]: This is correct. The lights do not have a default command. Lights are turned on and off only explicitly. There is no command that needs to continue running.

In the future, perhaps, lights would be controlled by a command.

**Comment by duane701** [comment]
PTAL

I have added the changes as a fixup! I will create a new branch with the fixed changes in place.

However, in the mean time you can review these changes as they are.

**Comment by duane701** [comment]
Replaced by #17.

**Pull request closed without merging**